commands/show-bl-hostname.go

```diff
@@ -40,7 +42,7 @@ func printBLHostnames(c *cli.Context) error {
 	res := resources.InitResources(c.String("config"))
 	res.DB.SelectDB(db)
 
-	data, err := getBlacklistedHostnameResultsView(res, "conn_count", 1000)
+	data, err := getBlacklistedHostnameResultsView(res, "conn_count", c.Int("limit"), c.Bool("no-limit"))
```

Comment:
If we have passed the config values into the res value do we need to parse the limit and no limit flags here? Since we should have those initialized could we use the res file below?

Reply:
Sorry about that, I was mistaken on how the command line arguments were handled. Ben pointed it out so this is all good!